search/templates/search/search.html

```diff
@@ -1,15 +1,62 @@
 {% extends 'base.html' %}
+
+{% block header %}
 {% load custom_tags %}
 {% load crispy_forms_tags %}
-
+{% load leaflet_tags %}
+{% leaflet_js %}
+{% leaflet_css %}
+{% endblock %}
 
 {%  block script %}
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
 <script>
     $(document).ready(function () {
         $("#complaints_button").click(function () {
             $("#complaints").toggle();
         });
     });
+
+    var markerList = [];
+    var mapCenter = [0,0];
+
+    {% for item in search_data.locations %}
```

Comment:
I think this `{% for item in search_data.locations %}` would create redundant javascript code to our result html file.

How about this? (I didn't run this code. Please take the brief idea from it)
```
let markerList = [];

const locations = [
{% for item in search_data.locations %}
[ {{ item.latitude }}, {{ item.longitude}}, "{{ item.google_map_url }}" ],
{% endfor %}
];

let sumLatitude = 0;
let sumLongitude = 0;

for (const loc of locations) {
  markerList.push(loc);
  sumLatitude += loc[0];
  sumLongitude += loc[1];
}

const mapCenter = [ sumLatitude / markerList.length, sumLongitude / makerList.length ];
```

Reply:
@alldne Good catch! Updated.